pkg/controller/statefulset/stateful_set.go

```diff
@@ -143,8 +143,8 @@ func (ssc *StatefulSetController) Run(workers int, stopCh <-chan struct{}) {
 	defer utilruntime.HandleCrash()
 	defer ssc.queue.ShutDown()
 
-	glog.Infof("Starting stateful set controller")
-	defer glog.Infof("Shutting down statefulset controller")
+	glog.Info("Starting StatefulSet controller")
```

Comment:
Approve of change to Info over Infof, but why are you rewriting the log message as well?

Reply:
![image](https://user-images.githubusercontent.com/25783555/31695889-b33b5332-b3df-11e7-80ef-540f7d0918ac.png)
yeah,maybe @enisoc think better use StatefulSet not stateful set,but I think in k8s file has more place use stateful set,now modify all file or leave it alone?thanks

Reply:
All other controllers use lower case in log statements. However, we are inconsistent with hyphenation, using two words, or appending them together. It should be one of "statefulset", "stateful-set", or "stateful set" but not "StatefulSet" for consistency.

Reply:
I suggested `StatefulSet` to follow the [docs style guide](https://kubernetes.io/docs/home/contribute/style-guide/#use-camel-case-for-api-objects). There's no rule that says log messages should follow the docs style guide, but it's the only documented style policy of any kind, so I thought it made sense to converge towards it.

Reply:
I think that we should be consistent so that when people are triaging controller_manager logs they aren't confused. I also agree that we should consider, at some point, moving all of the logging to follow the docs sytle guidelines.